core-api/src/test/java/com/optimizely/ab/optimizelyjson/OptimizelyJSONExtendedTest.java

```diff
@@ -0,0 +1,153 @@
+/**
+ *
+ *    Copyright 2020, Optimizely and contributors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package com.optimizely.ab.optimizelyjson;
+
+import com.optimizely.ab.config.parser.UnsupportedOperationException;
+import com.optimizely.ab.optimizelyjson.types.MD1;
+import com.optimizely.ab.optimizelyjson.types.MD2;
+import com.optimizely.ab.optimizelyjson.types.MD3;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class OptimizelyJSONExtendedTest extends OptimizelyJSONCoreTest {
```

Comment:
Instead of Core vs Extended you can use [assumeThat](https://junit.org/junit4/javadoc/4.12/org/junit/Assume.html#assumeThat(T,%20org.hamcrest.Matcher)) to conditionally ignore a parameterized variant.